source/amf-encoder.cpp

```diff
@@ -350,15 +350,15 @@ void Plugin::AMD::Encoder::SetVBVBufferStrictness(double_t v)
 			break;
 		}
 	}
-	strictBitrate = clamp(
+	strictBitrate = amf_clamp(
 		static_cast<uint64_t>(round(targetBitrate * ((double_t)m_FrameRate.second / (double_t)m_FrameRate.first))),
 		bitrateCaps.first, targetBitrate);
 
 	// Three-Point Linear Lerp
 	// 0% = looseBitrate, 50% = targetBitrate, 100% = strictBitrate
-	v                 = clamp(v, 0.0, 1.0);
-	double_t aFadeVal = clamp(v * 2.0, 0.0, 1.0);       // 0 - 0.5
-	double_t bFadeVal = clamp(v * 2.0 - 1.0, 0.0, 0.0); // 0.5 - 1.0
```

Comment:
Clamp seems like it should be from 0.0 to 1.0, not 0.0 to 0.0, but I'm not going to attempt to fix that.

Reply:
That should definitely be fixed, you can put it in this PR if you want to. It's supposed to be a 3-point lerp and used to be one at some point in the past, so it's best to restore that behavior.

Reply:
Alright, I'll change it.